core/src/main/java/de/danoeh/antennapod/core/widget/WidgetUpdater.java

```diff
@@ -36,6 +38,7 @@
  */
 public abstract class WidgetUpdater {
     private static final String TAG = "WidgetUpdater";
+    private static boolean showTimeLeft;
```

Comment:
Can this be declared as a local variable in the function where it is being used and not `static`?

Reply:
@tonytamsf Sure